xcode/config_generator/render_xcconfigs.rb

```diff
@@ -0,0 +1,98 @@
+require 'json'
+require 'mustache'
+
+
+# create config files if needed
+system("touch configs_data.json") unless File.exist?("configs_data")
+system("mkdir TargetConfigurations") unless Dir.exist?("TargetConfigurations")
+
+# call python script and generate configs to config file
+system("python gen_configurations.py > configs_data.json")
+
+
+# open settings + template file
+settings = JSON.load(File.open("settings.json"))
+target_xcconfig_tempate = File.read("target_xcconfig.mustache")
+
+
+# set global property
+targets = settings["targets"]
+
+# make tuple of key and value become mustache template element
+def config_option(key, value)
+    return { "key" => key, "value" => value }
+end
+
+# return empty array or generated dev team hash
+def development_team_if_needed(properties, account_type)
+    development_team_key = "DEVELOPMENT_TEAM"
+    if properties.key?(development_team_key)
+        return []
+    end
+
+    team_value = account_type == "Standard" ? "D4HA43V467" : "228J5MMU7S"
+    return [config_option(development_team_key, team_value)]
+end
+
+# return empty array or generated provisioning profile hash
+def provisioning_profile_if_needed(properties, account_type)
+    provisioning_key = "PROVISIONING_PROFILE_SPECIFIER"
+    if properties.key?(provisioning_key)
```

Comment:
copy-pasted piece of code

Reply:
how can I change it? This block of code like `guard` statement in swift, only check key if it exist